app.py

```diff
@@ -22,51 +22,74 @@
 import asyncio
 import time
 import logging
+
 from datetime import datetime
 
-from flask import Flask, Response
-from flask_apscheduler import APScheduler
+import responder
+import uvicorn
 
 from prometheus_client import generate_latest, CollectorRegistry, CONTENT_TYPE_LATEST
 
 from thoth.common import init_logging
 
-from thoth.metrics_exporter import __version__, config, thoth_package_version_total, thoth_package_version_seconds
-from thoth.metrics_exporter.jobs import get_thoth_solver_jobs, get_retrieve_unsolved_pypi_packages
-
+from thoth.metrics_exporter import __version__
+from thoth.metrics_exporter.jobs import (
+    get_solver_documents,
+    get_analyzer_documents,
+    get_janusgraph_v_and_e_total,
+    get_retrieve_unsolved_pypi_packages,
+    get_thoth_solver_jobs,
+)
 
-application = Flask(__name__)
-application.config.from_object(config.Config())
 
 init_logging()
 
-_LOGGER = logging.getLogger('thoth.metrics_exporter')
-_DEBUG = os.getenv('DEBUG', False)
+_LOGGER = logging.getLogger("thoth.metrics_exporter")
+_DEBUG = os.getenv("METRICS_EXPORTER_DEBUG", False)
+
+api = responder.API(title="Thoth Metrics Exporter", version=__version__)
+api.debug = _DEBUG
+
+
+@api.route(before_request=True)
+def prepare_response(req, resp):
+    """Just add my signature."""
+    resp.headers["X-Thoth-Metrics-Exporter-Version"] = f"v{__version__}"
 
 
-@application.route('/')
-def main():
+@api.route("/")
+async def main(req, resp):
     """Show this to humans."""
-    return "This service is not for humans!"
+    resp.text = "This service is not for humans!"
```

Comment:
Just an idea - what about redirecting to grafana (I think we don't have the url in deployment so its not that clear, right?)?

Reply:
+1 On the grafana, I coincidentally struggled to find out the grafana url few moments ago.

Reply:
done with https://github.com/thoth-station/metrics-exporter/pull/6/commits/7060984de178ff4a9c98183f8ad89df49c56c581